.gitignore

```diff
@@ -22,6 +22,7 @@ vendor/bundle/
 # OS garbage.
 Thumbs.db
 .DS_Store
+.directory
```

Comment:
Hi,
I am a free desktop user, thus I am using KDE Plasma as my desktop environment its file browser automatically creates this .directory file. It is common for all the KDE Plasma users. :-)
Thanks.